iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1569,7 +1569,7 @@
 
 "type.barrier.chain" = "Cadena";
 
-"type.barrier.city_wall" = "Muro de la ciudad";
+"type.barrier.city_wall" = "City Wall";
```

Comment:
Thanks! Can you please exclude this file from the commit? There's a bug that it reverses to en for some reason.